lizard_auth_server/models.py

```diff
@@ -450,3 +450,19 @@ def __unicode__(self):
         else:
             return "{role} in {org}".format(
                 role=self.role, org=self.organisation)
+
+    def as_dict(self):
+        """
+        """
+        org = Organisation.objects.get(id=self.organisation.id)
+        role = Role.objects.get(id=self.role.id)
+        portal = Portal.objects.get(id=role.portal.id)
+
+        return {
+            "for_all_users": self.for_all_users,
+            "portal_name": portal.name,
```

Comment:
for_all_users is puur iets van de server, mag niet naar de client (ander datamodel)